lib/utilities/aclUtils.js

```diff
@@ -114,7 +114,17 @@ aclUtils.parseAclXml = function parseAclXml(toBeParsed, log, next) {
         const jsonGrants = result
             .AccessControlPolicy.AccessControlList[0].Grant;
         log.trace('acl grants', { aclGrants: jsonGrants });
-        return next(null, jsonGrants);
+
+        if (!result.AccessControlPolicy.Owner
+            || result.AccessControlPolicy.Owner.length === 0
+            || result.AccessControlPolicy.Owner[0] === ''
+            || result.AccessControlPolicy.Owner[0].ID.length !== 1
```

Comment:
if Owner[0] is a string different from empty string, the worker will crash here :)